src/DeviceInput/Implementations/webDeviceInputSystem.ts

```diff
@@ -587,17 +597,18 @@ export class WebDeviceInputSystem implements IDeviceInputSystem {
             if (this.isDeviceAvailable(DeviceType.Touch)) {
                 const pointer = this._inputs[DeviceType.Touch];
 
-                for (const deviceSlotKey in pointer) {
-                    const deviceSlot = +deviceSlotKey;
+                // Get list of active touch ids and clear each one in the inputs array
+                for (const deviceSlotKey in Object.keys(this._activeTouchIds)) {
+                    const deviceSlot = this._activeTouchIds[+deviceSlotKey];

```

Comment:
Could we do the +deviceSlotKey only once ?

Reply:
I've changed code to just store +deviceSlotKey in a local var and use that throughout the blur code.